presto-spi/pom.xml

```diff
@@ -65,7 +65,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
```

Comment:
keep the test scope

Reply:
ImmutableList requires guava. Should I go back to using ArrayList?

Reply:
Use `unmodifiableList`/`ArrayList`. There are examples in that class.